src/domain/applications/queries.ts

```diff
@@ -1,6 +1,9 @@
 import { gql } from 'apollo-boost';
 
 export const BERTH_APPLICATIONS_QUERY = gql`
+  # Note:
+  # The operation name "BERTH_APPLICATIONS" is used as a string in other modules,
+  # for instance, as a value of refetchQueries; If you modify this name, consider changing it in other places too.
```

Comment:
Would there be a way to move the query name in an enum or a constant? That way the dependent refetchQueries would be linked to this query.

Reply:
Unfortunately not, it seems that the only way to get the query's name dynamically is using a `getOpertionName` method, which I found it [hidden in Stack Overflow](https://stackoverflow.com/a/55639277/9441362) with no mention in Apollo's documentation whatsoever.
I'll update my PR to use this method.